internal/pkg/deploy/cloudformation/stack/scheduled_job.go

```diff
@@ -68,6 +68,22 @@ const (
 	every = "@every "
 )
 
+type errScheduleInvalid struct {
+	reason error
+}
+
+func (e errScheduleInvalid) Error() string {
+	return fmt.Sprintf("schedule is not valid cron, rate, or preset: %s", e.reason.Error())
```

Comment:
nit:
```suggestion
	return fmt.Sprintf("schedule is not valid cron, rate, or preset: %v", e.reason)
```
this way if reason is nil then the code wouldn't panic :P . Unlikely, since it's private but just in case